### DISCUSSION TITLE
automatic deployment with oracle

### DISCUSSION THREAD

**Discussion by stellahansen2** [theory/sqitch#633] [General]
My team and I are trying to find a solution to automatically deploy oracle database objects (tables, stored procedures, data, etc.) from development to QA to production. I was wondering if anyone thought Sqitch would be a good tool, or if you believe you know a better one out there. Thank you so much for your time!

**Comment by theory** [comment]
I think you're likely to fine that those of us in the community _do_ like and recommend it over the alternatives. Do you have some more specific questions about features or functionality?

**Reply by stellahansen2** [reply]
I just wanted to make sure sqitch was compatible with oracle and had the functionalities to deploy tables, stored procedures, etc. I've looked into other platforms, but most don't have this specific capability.

**Reply by theory** [reply]
Yes, Sqitch is designed to run pure SQL\*Plus scripts, so pretty much whatever you can do in SQL\*Plus is supported by Sqitch. I suggest you start by skimming the [Oracle tutorial](http://sqitch.org/docs/manual/sqitchtutorial-oracle/).
